src/AppBundle/Controller/Api/SurveyController.php

```diff
@@ -0,0 +1,121 @@
+<?php
+
+namespace AppBundle\Controller\Api;
+
+use AppBundle\Entity\SurveyAnswer;
+use AppBundle\Entity\Survey;
+use AppBundle\Entity\SurveyQuestion;
+use Mcfedr\JsonFormBundle\Controller\JsonController;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\ParamConverter;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Symfony\Component\HttpFoundation\Request;
+
+class SurveyController extends JsonController
+{
+    /**
+     * @Route("/surveys", name="api_surveys")
+     * @Method("GET")
+     */
+    public function apiSurveysAction()
+    {
+        $user = $this->getUser();
+        if (!$user) {
+            return $this->json(['message' => 'User is not authorized'], 401);
+        }
+        $em = $this->getDoctrine()->getManager();
+        $surveys = $em->getRepository(Survey::class)->findSurveyByUser($user);
+        if (!$surveys) {
+            return $this->json(['message' => 'No surveys'], 404);
+        }
+        $serializer = $this->get('serializer');
+
+        $json = $serializer->normalize(
+            $surveys,
+            null,
+            array('groups' => array('group1'))
+        );
+
+        return $this->json(['surveys' => $json], 200);
+    }
+
+    /**
+     * @param Survey $survey
+     * @Route("/surveys/{id}", name="api_survey")
+     * @Method("GET")
+     * @ParamConverter("survey", class="AppBundle:Survey")
+     */
+    public function apiSurveyAction(Survey $survey)
+    {
+        $user = $this->getUser();
+        if (!$user) {
+            return $this->json(['message' => 'User is not authorized'], 401);
+        }
+
+        if (!$survey) {
+            return $this->json(['message' => 'No surveys'], 404);
+        }
+        $serializer = $this->get('serializer');
+        $jsonSurvey = $serializer->normalize(
+            $survey,
+            null,
+            array('groups' => array('group1', 'group2'))
+        );
+        if ($survey->getStatus() == 'submited') {
+            $answers = $survey->getAnswers();
```

Comment:
For answers must be other resource or they should included for response of servey